packages/web3-eth/test/integration/defaults.transactionBlockTimeout.test.ts

```diff
@@ -19,62 +19,60 @@ import { DEFAULT_RETURN_FORMAT } from 'web3-utils';
 import { Web3PromiEvent } from 'web3-core';
 import { TransactionReceipt } from 'web3-types';
 import { TransactionBlockTimeoutError } from 'web3-errors';
-import { SendTransactionEvents, Web3Eth } from '../../src';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Web3 } from 'web3';
```

Comment:
I think it is better to keep using `Web3Eth` except if there is a strong reason for not doing so.

Reply:
To use a local wallet, we need to use web3. because web3-eth doesn't have functionality to sign transactions from wallet